QuickBrick/plugins/TViOSLogin/src/Common/Utils/index.js

```diff
@@ -27,3 +30,48 @@ export async function setToLocalStorage(key, value) {
 export async function getFromLocalStorage(key) {
   return storage.getItem(key);
 }
+
+function validateStyles(pluginData) {
+  const keys = Object.keys(pluginData);
+  keys.forEach((key) => {
+    const type = key.split('_').pop();
+    if (type === 'fontsize' || type === 'fontcolor') {
+      validateKey(type, key, pluginData);
+    }
+  });
+}
+
+function validateKey(type, key, pluginData) {
+  const keysValidation = {
+    fontsize: validateFontsize,
+    fontcolor: validateFontcolor
+  };
+
+  return keysValidation[type](key, pluginData);
+}
+
+const validateFontsize = (key, pluginData) => {
+  const value = pluginData[key];
```

Comment:
if pluginData is undefined js crash

Reply:
pluginData is an empty object by default that is defined in getPluginData function
```js
export function getPluginData(screenData) {
  let pluginData = {};

  if (screenData && screenData.general) {
    pluginData = { ...pluginData, ...screenData.general };
    validateStyles(pluginData);
  }

  return pluginData;
}
```